### PR TITLE
Ping authentication endpoint, docs

### PR DESCRIPTION
**Pull request by pkopac**
Introducing the test auth endpoint `Ping.ping()`.
If no other fixes/features will come, this will be release 1.0.1

### CODE REVIEW

File: test/chartmogul/ping.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            'message': 'No valid API key provided',
+            'param': null
+        };
+        nock(config.API_BASE)
+            .get('/v1/ping')
+            .reply(401, errorMsg);
+
+        return Ping.ping(config)
+            .then(res => {
+                fail("Shouldn't succeed!");
```
Comment (bilbof) [inline test/chartmogul/ping.js]: Is `fail` defined?

**Comment by bilbof** [comment]
Looks like you need a different version of eslint-config-standard and eslint-plugin-standard.

```
npm ERR! peerinvalid The package eslint@2.13.1 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer eslint-config-standard@5.3.5 wants eslint@>=2.0.0
npm ERR! peerinvalid Peer eslint-plugin-standard@2.0.1 wants eslint@>=3.0.0
```

https://travis-ci.org/chartmogul/chartmogul-node/jobs/200371509#L323

File: test/chartmogul/ping.js (hunk 1 above)
Comment (pkopac) [inline test/chartmogul/ping.js]: yes, it's part of mocha. Like `describe` or `it`. `eslint` didn't know it though. And I forgot to run `eslint`.

**Comment by pkopac** [comment]
@bilbof stupid of me to rely on automatic version resolution, indeed. Also newest versions don't work on older node.js, so I've reverted to original versions, I hope it'll build now.

**Pull request merged**